CRM/Import/Parser.php

```diff
@@ -1626,6 +1626,11 @@ protected function getContactMatchingFields(): array {
         $fields[$value] = $contactFields[$value] ?? NULL;
         $title = $fields[$value]['title'] . ' (match to contact)';
         $fields[$value]['title'] = $title;
+        $fields[$value]['entity'] = 'Contact';
+        if ($customFieldId) {
+          $fieldSpec = CRM_Core_BAO_CustomField::getField($customFieldId);
+          $fields[$value]['apiv4_name'] = $fieldSpec['custom_group']['name'] . '.' . $fieldSpec['name'];
```

Comment:
There's a function for that:
```suggestion
          $fields[$value]['apiv4_name'] = CRM_Core_BAO_CustomField::getLongNameFromShortName($value);
```